Valeriia/Movies_database_project/src/pages/HomePage/HomePage.tsx

```diff
@@ -73,13 +88,21 @@ class HomePage extends Component<MoviesProps, IMoviesState> {
         <SortPannel
           moviesCount={this.props.movies.length}
           onClickSortBy={this.onClickSortByHandler}
-          sortBy={this.props.sortBy}
-        />
-        <Movies
-          movies={this.props.movies}
-          isLoading={this.props.isLoading}
-          isError={this.props.isError}
+          sortBy={this.state.sortBy}
         />
+
+        <InfiniteScroll
+          dataLength={this.props.movies.length}
+          next={() => this.fetchData(this.props.movies.length + 9, true)}
+          hasMore={true}
```

Comment:
Eventually movies will end and you will get an error, so you have to write a check here. Make sure that total > this.props.movies.length;